### PR TITLE
Support using existing secrets with cloud services

### PR DESCRIPTION
**Pull request by FalconerTC**
Right now, supplying `postgresql.endpoint.cloudservice` requires the postgres password to be from values instead of supporting K8s secrets. Using cloudservice shouldn't prevent you from being able to use k8s secrets to store the credentials. This follows conventions seen often in other charts. The same applies to rabbitmq.

These changes are not breaking and we are currently deploying with them

### CODE REVIEW

**Comment by HardNorth** [comment]
@FalconerTC Thank you for the patch! I agree with your point. But I need to double check the patch to ensure nothing left unpatched, especially in new deployments. I'll do that after my current vacation (after July 25th).

File: reportportal/templates/analyzer-statefulset.yaml [hunk 1]
```diff
@@ -44,13 +44,13 @@ spec:
        {{ end }}
        {{ end }}
         - name: RP_AMQP_PASS
-        {{ if .Values.rabbitmq.endpoint.cloudservice }}
```
Comment (HardNorth) [inline reportportal/templates/analyzer-statefulset.yaml:47]: Could you please also delete this flag from `values.yaml`?

File: reportportal/templates/api-deployment.yaml [hunk 2]
```diff
@@ -77,13 +77,13 @@ spec:
         - name: RP_DB_USER
           value: "{{ .Values.postgresql.endpoint.user }}"
         - name: RP_DB_PASS
-        {{ if .Values.postgresql.endpoint.cloudservice }}
```
Comment (HardNorth) [inline reportportal/templates/api-deployment.yaml:80]: And delete this one too?

**Review by HardNorth — CHANGES_REQUESTED** [review]

File: reportportal/templates/api-deployment.yaml (hunk 2 above)
Comment (FalconerTC) [reply reportportal/templates/api-deployment.yaml:80]: @HardNorth Any chance for a re-review?

Comment (HardNorth) [reply reportportal/templates/api-deployment.yaml:80]: @FalconerTC please resolve conflicts

**Comment by HardNorth** [comment]
@FalconerTC ping

**Review by HardNorth — APPROVED** [review]

**Pull request merged**